src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.cs

```diff
@@ -30,7 +30,7 @@ public partial struct CounterSample
         public static bool operator ==(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
         public static bool operator !=(System.Diagnostics.CounterSample a, System.Diagnostics.CounterSample b) { throw null; }
     }
-    public sealed partial class PerformanceCounter
+    public sealed partial class PerformanceCounter : System.ComponentModel.Component, System.ComponentModel.ISupportInitialize
```

Comment:
There is none currently. I discussed with @danmosemsft and @joperezr. As ApiCompat logged errors for netfx differences there were two options:
- Remove the entire project as it is just a ref
- Adjust the refs to match the netfx contract

We decided to take the latter as the refs are ok as they are. They don't get packaged, just build